vehicle-model-lifecycle/requirements.txt

```diff
@@ -1,4 +1,4 @@
-velocitas-lib @ git+https://github.com/eclipse-velocitas/velocitas-lib@v0.0.3
+velocitas-lib @ git+https://github.com/eclipse-velocitas/velocitas-lib@v0.0.2
```

Comment:
why did you downgrade this?

Reply:
seems I used outdated version of vehicle-app-python-template to test :) changes reverted